src/client/components/app/Transactions/DepositTx.tsx

```diff
@@ -137,14 +150,16 @@ export const DepositTx: FC<DepositTxProps> = ({
       return;
     }
 
-    dispatch(
-      VaultsActions.getExpectedTransactionOutcome({
-        transactionType: 'DEPOSIT',
-        sourceTokenAddress: selectedSellTokenAddress,
-        sourceTokenAmount: toWei(debouncedAmount, selectedSellToken.decimals),
-        targetTokenAddress: selectedVault.address,
-      })
-    );
+    if (simulationsEnabled) {
```

Comment:
nitY I always think it's cleaner to just early return than checking where the curly brace ends

```suggestion
    if (!simulationsEnabled) {
        return;
    }
```

Reply:
It can be done if I also change the order of the action dispatched after this statement. Thought it was clearer just to place it as it is. Guess it is matter of preference on this one.

Reply:
Happy to leave it as is